app/controllers/application_controller.rb

```diff
@@ -8,6 +8,10 @@
 require 'aes_256_cbc_encryptor'
 
 class ApplicationController < ActionController::API
+  include ActionController::RequestForgeryProtection
+  protect_from_forgery with: LoggingForgeryStrategy, if: -> { ActionController::Base.allow_forgery_protection }
```

Comment:
`LoggingForgeryStrategy` is a temporary "strategy" for handling violations.  That being, logging them to Sentry for evaluation.
Will change this to `with: :exception` when ready to turn it on, which will raise `ActionController::InvalidAuthenticityToken` exception.

Reply:
This looks really clean.  Does this mean you need to create a LoggingForgeryStrategy?